src/data/finset.lean

```diff
@@ -1489,6 +1521,15 @@ have ∀b a, a ∈ s → b ∈ t₁ a → (∃ (a : α), a ∈ s ∧ b ∈ t₂
   from assume b a ha hb, ⟨a, ha, finset.mem_of_subset (h a ha) hb⟩,
 by simpa only [subset_iff, mem_bind, exists_imp_distrib, and_imp, exists_prop]
 
+lemma bind_sub_bind_of_sub_left {α : Type*} {s₁ s₂ : finset α}
```

Comment:
Use `subset` instead of `sub` in the name: normally, `sub` refers to subtraction.